app/controllers/SignUpController.scala

```diff
@@ -51,8 +54,20 @@ class SignUpController @Inject() (
             for {
               user <- userService.save(user)
               authInfo <- authInfoRepository.add(loginInfo, authInfo)
```

Comment:
Just noticing: should we try to be consistent naming unused variables, as in `SocialAuthController?